spec/javascripts/components/reorderable-list-spec.js

```diff
@@ -117,9 +117,9 @@ describe('Reorderable list component', function () {
     var reorderableList
     var sortable
     beforeEach(function () {
-      reorderableList = new GOVUK.Modules.ReorderableList()
-      reorderableList.start($(element))
-      spyOnEvent($(element), 'reorder-drag')
+      reorderableList = new GOVUK.Modules.ReorderableList(element)
+      reorderableList.init()
```

Comment:
In other places the initialisation of the module is done in a single line e.g. `new GOVUK.Modules.ReorderableList(element).init()`. I can see that in some places (in this file, for example) that this is done so that a spy can be initialised, but here it looks like it could be combined. Could this be made more consistent?

Reply:
Yes, I've tried to only separate it for Spys – this might be accidental, will double-check and update.

Reply:
Ah, we need the instance on [L123](https://github.com/alphagov/govuk_publishing_components/pull/2095/files/a466858544bc561805e4dbc2bb2c63cb8c9bd8af..35aeb30eabe389109b48fc9fb8705b979de6326a#diff-d528e7ff346f0e72e2a2ebfe3b55d754ebc8f60603c04eb3c4aff0349938e73eR123) to check for a property, that's why I had to keep it separately.